2024/political-donations-top-pacs_bubble/index.html

```diff
@@ -0,0 +1,43 @@
+<html lang="en">
+    <head>
+        <title>Top PACs By Donations</title>
+        <!-- PLACE ALL REPOSITORY CSS FILES IN A BRACKET LIKE THIS -->
+        <!-- build:css inline -->
+        <link rel="stylesheet" href="style.css" />
+        <!-- /build -->
+    </head>
+
+    <body>
+        <div id="chart-div1"></div>
+        <div id="chart-div2"></div>
```

Comment:
If you're gonna put two visuals in the same page, you'll need to make it so that it's responsive: https://codepen.io/sandeepvaman/pen/wxgbZL

Reply:
tbh idk what i was doing here and decided not to sure this one so just removing